docs/howto/operate/move-hub.md

```diff
@@ -23,29 +23,67 @@ home directory storage!
 
 Primarily used with GKE right now.
 
-1. In the target NFS server, create a new ssh key-pair, with
+1. SSH into **both** the target server and source server and then enable the `ubuntu` user on **both** servers with `sudo su ubuntu`.
+   This will ensure that the keys we are about to create will be assigned to the correct user.
+2. In the target NFS server, create a new ssh key-pair, with
    `ssh-keygen -f nfs-transfer-key`
-
-2. Copy the public key `nfs-transfer-key.pub` to the source NFS
-   server's `/home/ubuntu/.ssh/authorized_keys`, so the target
-   NFS server can `scp` from the source NFS server.
-
-3. Copy the NFS home directories from the source NFS server to
+3. Append the public key `nfs-transfer-key.pub` to the source NFS
+   server's `/home/ubuntu/.ssh/authorized_keys` file, so the target
+   NFS server can open SSH connections to the source NFS server.
```

Comment:
What do you think about splitting the sentence? (I read it 2 times until I made a visual model and was sure I understood :sweat_smile: )
```suggestion
3. Append the public key `nfs-transfer-key.pub` to the source NFS
   server's `/home/ubuntu/.ssh/authorized_keys` file. This way, the target
   NFS server will be able to open SSH connections to the source NFS server.
```